packages/experimental-utils/src/ts-eslint/Rule.ts

```diff
@@ -211,6 +211,12 @@ interface RuleContext<
    */
   getDeclaredVariables(node: TSESTree.Node): Scope.Variable[];
 
+  /**
+   * returns the cwd passed to Linter.
+   * It is a path to a directory that should be considered as the current working directory.
+   */
+  getCwd(): string;
```

Comment:
```suggestion
  /**
   * Returns the current working directory passed to Linter.
   * It is a path to a directory that should be considered as the current working directory.
   * This was added in v6.6.0
   * @since 6.6.0
   */
  getCwd?(): string;
```